logging/cluster-logging-enabling-json-logging.adoc

```diff
@@ -0,0 +1,21 @@
+:context: cluster-logging-enabling-json-logging
+[id="cluster-logging-enabling-json-logging"]
+= Enabling JSON logging
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+You can configure the Cluster Log Forwarder custom resource (CR) to forward structured JSON log entries as JSON objects in JSON output records.
+
+[IMPORTANT]
+====
+Before using this feature with Elasticsearch, see xref:../logging/cluster-logging-enabling-json-logging.adoc#cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_cluster-logging-enabling-json-logging[Configuration of JSON log data for default Elasticsearch] and xref:../logging/cluster-logging-enabling-json-logging.adoc#cluster-logging-forwarding-json-logs-to-the-default-elasticsearch_cluster-logging-enabling-json-logging[Forwarding JSON logs to the default Elasticsearch log store].
```

Comment:
I'm not sure that "See" is the verb you want here?  I think what you're trying to say is something like "Before you can use this feature with Elasticsearch, you must first complete....  [two links]"?